c/include/cuvs/neighbors/cagra.h

```diff
@@ -34,7 +34,9 @@ enum cuvsCagraGraphBuildAlgo {
   /* Experimental, use NN-Descent to build all-neighbors knn graph */
   NN_DESCENT = 2,
   /* Experimental, use iterative cagra search and optimize to build the knn graph */
-  ITERATIVE_CAGRA_SEARCH = 3
+  ITERATIVE_CAGRA_SEARCH = 3,
+  /* Use ACE (Augmented Core Extraction) to build the graph */
```

Comment:
Can we point to a section of the docs (maybe the params?) where a user can learn more about the ACE algorithm?